File: study/src/main/java/com/grasstudy/study/event/scheme/AttendEvent.java
package com.grasstudy.study.event.scheme;

import com.grasstudy.attend.entity.Attend;
import com.grasstudy.common.event.EventMessage;
import lombok.*;

@Getter
@Builder
@ToString
@NoArgsConstructor(access = AccessLevel.PRIVATE)
@AllArgsConstructor(access = AccessLevel.PRIVATE)
public class AttendEvent implements EventMessage {

	private Attend attend;
}

File: noticer/src/main/kotlin/com/grasstudy/noticer/event/attend/scheme/AttendEvent.kt
package com.grasstudy.noticer.event.attend.scheme

data class AttendEvent(val studyId: String, val userId: String, val state: AttendState) {
	override fun toString(): String {
		return "AttendEvent(studyId='$studyId', userId='$userId', state=$state)"
	}
}
